.github/labeler.yml

```diff
@@ -1,46 +1,53 @@
-# Add 'Multicast' label to any root file changes
-Multicast:
-  - setup.py
-  - multicast/*.py
+# Add 'Python Repo' label to any root file changes
+Python Repo:
+  - changed-files:
+    - any-glob-to-any-file: 'setup.py'
+    - any-glob-to-any-file: 'pythonrepo/*.py'
 
 # Add 'Lint' label to any lint file changes
 Linter:
-  - .stickler.yml
-  - .hound.yml
-  - .yamllint.conf
-  - .flake8.ini
-  - .deepsource.toml
-  - tests/check_*
+  - changed-files:
+    - any-glob-to-any-file: '.stickler.yml'
+    - any-glob-to-any-file: '.hound.yml'
+    - any-glob-to-any-file: '.yamllint.conf'
+    - any-glob-to-any-file: '.flake8.ini'
+    - any-glob-to-any-file: '.deepsource.toml'
+    - any-glob-to-any-file: 'tests/check_*'
 
 # Add 'Lang' label to any python file changes
 Python Lang:
```

Comment:
It is unclear if the whitespace here needs quoted from the docs:
 - see https://github.com/actions/labeler/blob/506e1a00c6c05ba98ac585056a23f16073fa5915/README.md for details